### PR TITLE
Use randomness not entropy

### PR DESCRIPTION
**Pull request by richsalz**
This PR does not touch the crypto/rand directory which is being handled separately.

### CODE REVIEW

File: doc/man3/RAND_add.pod [hunk 1]
```diff
@@ -27,10 +27,10 @@ if the data at B<buf> are unpredictable to an adversary, this
 increases the uncertainty about the state and makes the PRNG output
 less predictable. Suitable input comes from user interaction (random
 key presses, mouse movements) and certain hardware events. The
-B<entropy> argument is (the lower bound of) an estimate of how much
-randomness is contained in B<buf>, measured in bytes. Details about
-sources of randomness and how to estimate their entropy can be found
-in the literature, e.g. RFC 1750.
+B<randomness> argument is an estimate of how much randomness is contained in
+B<buf> and should be a nujbmer between zero and B<num>.
```
Comment (kroeckx) [inline doc/man3/RAND_add.pod]: number

Comment (kroeckx) [inline doc/man3/RAND_add.pod]: It's no longer saying that it's in bytes, it might also be expressed in bits.

File: doc/man3/RAND_bytes.pod [hunk 2]
```diff
@@ -24,8 +24,8 @@ enough randomness to ensure an unpredictable byte sequence.
 
 RAND_pseudo_bytes() has been deprecated; use RAND_bytes() instead.
 
-The contents of B<buf> is mixed into the entropy pool before retrieving
-the new pseudo-random bytes unless disabled at compile time (see FAQ).
+The contents of B<buf> is mixed into the randomness pool before retrieving
+the new pseudo-random bytes.
```
Comment (kroeckx) [inline doc/man3/RAND_bytes.pod]: This is actually no longer the case.

**Comment by richsalz** [comment]
Added a commit that addresses your feedback so far, @kroeckx .

**Comment by richsalz** [comment]
ping.  i have some more doc fixes to do after this.

File: apps/app_rand.c [hunk 3]
```diff
@@ -93,13 +93,14 @@ int app_RAND_write_file(const char *file)
 {
     char buffer[200];
 
-    if (egdsocket || !seeded)
+    if (egdsocket || !seeded) {
         /*
          * If we did not manage to read the seed file, we should not write a
-         * low-entropy seed file back -- it would suppress a crucial warning
+         * low-random seed file back -- it would suppress a crucial warning
```
Comment (kaduk) [inline apps/app_rand.c]: "low-random" doesn't parse as English.

File: doc/man3/RAND_egd.pod [hunk 4]
```diff
@@ -36,28 +36,28 @@ OpenSSL built-in PRNG using L<RAND_add(3)>.
 
 =head1 NOTES
 
-On systems without /dev/*random devices providing entropy from the kernel,
-the EGD entropy gathering daemon can be used to collect entropy. It provides
-a socket interface through which entropy can be gathered in chunks up to
+On systems without /dev/*random devices providing randomness from the kernel,
+EGD. It provides
```
Comment (kaduk) [inline doc/man3/RAND_egd.pod]: This is not a complete sentence.

**Comment by richsalz** [comment]
Added commit with @kaduk's comments addressed.

File: apps/app_rand.c [hunk 5]
```diff
@@ -95,9 +95,9 @@ int app_RAND_write_file(const char *file)
 
     if (egdsocket || !seeded) {
         /*
-         * If we did not manage to read the seed file, we should not write a
-         * low-random seed file back -- it would suppress a crucial warning
-         * the next time we want to use it.
+         * If we didn't manage to read the seed file, don't write a
+         * file out -- it would suppress a crucial warning the next
```
Comment (kaduk) [inline apps/app_rand.c]: side note: FreeBSD documentation style guide says to not use contractions.  But of course we can have our own style.

**Comment by richsalz** [comment]
thanks.
side-note: documentation, not code comments. :)
but now you've made me look at it.  new PR coming up I think.  It's a change from RAND work.

**Pull request closed without merging**